src/completers/git.zsh

```diff
@@ -58,24 +58,26 @@ PREVIEW_OPTIONS
 
 _fzf_complete_git() {
     setopt local_options extended_glob
-    local arguments=$@
+    local arguments=("${(Q)${(z)@}[@]}")
```

Comment:
Withi this fix, `(Q)` on line 85, 140, 234, 239, 273, 291, 679 are no longer necessary.